test/kwokctl/kwokctl_port_forward.test.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
```

Comment:
Rename this to `kwokctl_port_forward_test.sh`,
and create `kwokctl_binary_port_forward.test.sh` as the entry point.

For each runtime, create something like this
```
DIR="$(dirname "${BASH_SOURCE[0]}")"

DIR="$(realpath "${DIR}")"

source "${DIR}/helper.sh"

function main() {
  local all_releases=("${@}")

  test_all "binary" "port_forward" "${all_releases[@]}" || exit 1
}

requirements_for_binary

main $(supported_releases)
```

Reply:
https://github.com/kubernetes-sigs/kwok/blob/d0b443fe607a1c26108bce99dd12c4d4e3231e6b/.github/workflows/test.yaml#L213-L216

also need to add

``` yaml
      - name: Test Forward Port
        shell: bash
        run: |
          ./hack/e2e-test.sh kwokctl/kwokctl_${{ matrix.kwokctl-runtime }}_forward_port
```